bin/node/runtime/src/constants.rs

```diff
@@ -11,6 +16,268 @@ pub mod currency {
     lazy_static::lazy_static! {
         /// PLASM GENESIS TOKEN HODERS
         pub static ref HOLDERS: Vec<(AccountId, Balance)> = vec![
+            (MultiSigner::from(Public::from_full(&hex!["1f57f692ecc6c7031f7b89e283a14a111a273ef5ca4295ed114cef5faed3f1a66da013f76b420d7a054aec6e76ecc077a70dc9951dc41b1679e20d1d9eb665b0"][..]).unwrap()).into_account(), 216982973545339075380)
```

Comment:
How about turning this into a function?
`MultiSigner::from(Public::from_full(&hex!["address"][..]).unwrap()).into_account()`

Reply:
Ok, no problem